qark/qarkMain.py

```diff
@@ -410,15 +411,15 @@ def setup_argparse():
     return parser
 
 
-def run_automated_defaults(pathToReport, pathToApk):
+def run_automated_defaults(pathToApk, pathToReport):
     common.args = argparse.Namespace()
     common.args.exploit = 0
     common.args.install = 0
     common.args.source = 1
     common.args.reportDir = pathToReport
-    common.args.reportdir = pathToReport
     common.args.apkpath = pathToApk
     common.args.debuglevel = None
+    common.args.decompile = 0
```

Comment:
Need a default here for when the main() determines if we should perform static code analysis, while in non-interactive mode. Without the default, it tries to check the decompile argument (which doesn't exist).

I would argue that the default should be "false", which will cause static code analysis to occur by default. This is a primary reason for running this without interactive mode.